cpp/tests/io/parquet_test.cpp

```diff
@@ -3323,4 +3323,109 @@ TEST_F(ParquetWriterTest, EmptyListWithStruct)
   cudf::test::expect_columns_equal(result.tbl->view().column(0), *L0);
 }
 
+class ThriftReaderLite {
```

Comment:
We have a thrift reader here https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/src/io/parquet/compact_protocol_reader.hpp. Can you please include the header `#include <src/io/parquet/compact_protocol_reader.hpp>` in this file and use that one instead?

Reply:
@PointKernel  will do.  Yes, I stole that from compact_protocol_reader anyway, but wasn't sure I could include that in the test code.

Reply:
done!